css/style.css

```diff
@@ -1,7 +1,78 @@
-.book {
-  color: blue;
+* {
+  margin: 0;
+  padding: 0;
 }
 
-.container {
-  text-align: center;
+body {
+  background-color: antiquewhite;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  margin: 30px 0;
+}
+
+.book-span {
+  color: rgb(3, 3, 114);
+}
+
+.form-section {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input[type="text"] {
+  border: lightsteelblue;
+  width: 40%;
+  margin: 20px 0;
+  background-color: white;
+  height: 40px;
+  margin-left: 10%;
+}
+
+input[type="button"] {
+  background-color: lightskyblue;
+  width: 60%;
+  height: 40px;
+  border: none;
+  margin: 20px 0;
+  color: white;
+  font-size: 16px;
+}
+
+table {
+  background-color: aliceblue;
+  width: 60%;
+  margin-left: 10%;
+}
+
+.books {
+  background-color: aqua;
+  width: 100%;
+}
```

Comment:
- [ ] Kindly make your design match the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) 👍🏿